.gitignore

```diff
@@ -38,12 +38,13 @@ themes/*/config/settings_*.json
 
 modules/*
 override/*
+/themes/StarterTheme
```

Comment:
Philosophically it should be, but I'd rather prevent someone committing it without noticing.